app/models/candidate_interface/personal_details_form.rb

```diff
@@ -8,5 +8,9 @@ class PersonalDetailsForm
     def name
       "#{first_name} #{last_name}"
     end
+
+    def date_of_birth_format
+      DateTime.new(year.to_i, month.to_i, day.to_i).strftime('%e %B %Y')
```

Comment:
Shall we make it a date format? https://thoughtbot.com/blog/custom-formats-for-datetime

Reply:
The name of this method implies it returns a `format` — how about just `date_of_birth`?

Reply:
`date_of_birth` was the initial stab, but it seems to cause our form library to blow up here:

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/37912bf980b102de73a6377113418e9ff85507ba/app/views/candidate_interface/personal_details/edit.html.erb#L20

We both were kind of scratching our heads as to why.

In a way, I think it's better if this method (which returns a string, not a `Date`) follows some kind of naming like `formatted_date_of_birth` or the one being used here. But I don't have strong opinions... :shipit:

Reply:
The `govuk_date_field` helper expects a `Date` attribute but it _should_ work with any object that responds to `#day`, `#month` and `#year`, see [here](https://github.com/DFE-Digital/govuk_design_system_formbuilder/blob/master/lib/govuk_design_system_formbuilder/elements/date.rb#L39).

As always, if there's a better way of handling this I'm open to suggestions

Reply:
@peteryates I'm changing to a `Date` attribute in a soon to be submitted PR 👍